src/server/pbsd_init.c

```diff
@@ -256,14 +256,18 @@ init_server_attrs()
 {
 	resource_def    *prdef = NULL;
 	resource        *presc = NULL;
-	struct attribute attrib = {0};
-	int i = 0;
+	attribute 	temp_attr;
+	struct 		attribute attrib = {0};
```

Comment:
In attribute.h we have... typedef struct attribute attribute;
Normally, (for the sake of clarity) this would be... typedef struct attribute attribute_t;
You've used both "attribute" and "struct attribute" here, even though they are the same thing. Please use one or the other.

Reply:
@mike0042,
I agree with you that we could have actually defined it as struct attribute attribute_t. I am using attribute only in both the places.
Since I have now created a separate function to set value for any attribute through server this temp_attr is not required here.(But defined inside the function set_attr_svr)